src/decorators.ts

```diff
@@ -52,3 +54,27 @@ function getOpenAPIMetadata(target: object, key: string): OpenAPIParam {
 function setOpenAPIMetadata(value: OpenAPIParam, target: object, key: string) {
   return Reflect.defineMetadata(OPEN_API_KEY, value, target.constructor, key)
 }
+
+/**
+ * Supplement action with response body type annotation.
+ *
+ * @param responseType Class annotated with class-validator annotations
+ */
+export function ResponseBody(responseClass: any, statusCode: number = 200) {
```

Comment:
unhappy about the "any" type for passed class. If you have some better idea how to ensure a class-validator-enabled or even a normal class is passed I'd be happy to learn something